modules/rest-compatibility/src/main/java/org/elasticsearch/rest/compat/version7/RestGetActionV7.java

```diff
@@ -33,10 +33,11 @@
 import static org.elasticsearch.rest.RestRequest.Method.HEAD;
 
 public class RestGetActionV7 extends RestGetAction {
-
-    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(LogManager.getLogger(RestGetAction.class));
     static final String TYPES_DEPRECATION_MESSAGE = "[types removal] Specifying types in "
         + "document get requests is deprecated, use the /{index}/_doc/{id} endpoint instead.";
+    static final String TYPED_API_COMPATIBLE_WARNING = "Using typed version of Get API. Use /{index}/_doc/{id} instead.";
```

Comment:
Can this (and all other compatible warnings) be a past tense message that explains that what compatibly did ?

For this one...something like:
```
HTTP compatibility was applied to this request. The GET request for /<index-name>/<type-name>/<id> was mapped to /<index-name>/_doc/<id>. 
```
Where the actual index-name, type, and id are emitted to the log.